integration/currentop_test.go

```diff
@@ -33,6 +33,9 @@ import (
 )
 
 func TestCurrentOpGetMore(tt *testing.T) {
+	// TODO https://github.com/FerretDB/FerretDB/issues/4871
+	tt.Skip("https://github.com/FerretDB/FerretDB/issues/4871")
```

Comment:
This `TestCurrentOpGetMore` test fails on CI sometimes, while another `TestCurrentOpExplain` seems stable as I haven't spotted CI failure.